inlong-sort/sort-flink/sort-flink-v1.15/sort-connectors/iceberg/src/main/java/org/apache/inlong/sort/iceberg/sink/IcebergStreamWriterMetrics.java

```diff
@@ -109,14 +109,26 @@ void flushDuration(long flushDurationMs) {
     }
 
     void outputMetricsWithEstimate(int size, long time) {
-        if (sourceMetricData != null) {
-            sourceMetricData.outputMetrics(1, size, time);
+        if (sinkExactlyMetric != null) {
+            sinkExactlyMetric.invokeWithId(1, size, time);
```

Comment:
what is Id here ? can't tell it is checkpoint id by method name. just invoke is fine since the class name is exactly metric

Reply:
I forgot to rename it, fixed it, thx~